ThreadJob.psd1

```diff
@@ -8,7 +8,7 @@
 RootModule = '.\ThreadJob.dll'
 
 # Version number of this module.
-ModuleVersion = '1.1.0'
+ModuleVersion = '1.1.1'

```

Comment:
FYI. Looks like build files in the WIP PR in PS repo use version="1.1.0", so that will need to be updated.